middleware.ts

```diff
@@ -0,0 +1,14 @@
+import { NextResponse } from "next/server";
+import type { NextRequest } from "next/server";
+
+export function middleware(request: NextRequest) {
+  // Check if the user is trying to access the root "/"
+  if (request.nextUrl.pathname === "/") {
```

Comment:
Use const for immutable values – Since request.nextUrl.pathname does not change, store it in a constant.
```suggestion
  const { pathname } = request.nextUrl;

  if (pathname === "/") {
```

Reply:
done